### ISSUE TITLE
Dashboard listens for wrong AuthorizationIncrease events

### ISSUE THREAD

**Issue by cygnusv** [threshold-network/token-dashboard#626]
The consensus we had with @pdyraga and @vzotova was that, for authorization purposes, we'd look for events only in the Staking contract, since applications may have different internal semantics. See https://github.com/threshold-network/solidity-contracts/issues/88#issuecomment-1111059312

However, during the work for #603, we've detected that the dashboard listens for events on the application:
https://github.com/threshold-network/token-dashboard/blob/bc59c7ce2169ef9c376bb2086d556599970b8a87/src/hooks/staking-applications/useSubscribeToAuthorizationIncreasedEvent.ts#L12-L27

Coincidentally, tBTC and RB app contracts define `AuthorizationIncreased` events with the exact signature than those from the staking contract, so for their particular case, listening for app events work. However, this is problematic for the TACo app contract that doesn't define them in the same way (which shouldn't be a problem)

The solution would be to listen for these events in the staking contract, as originally intended.

**Comment by michalsmiarowski** [comment]
@cygnusv Based on @pdyraga [comment](https://github.com/threshold-network/solidity-contracts/issues/88#issuecomment-1111059312) it looks like we can also change the source of `AuthorizationDecreaseRequested` and `AuthorizationDecreaseApproved` events to `TokenStaking` contract.  The only problem is that in the dashboard for `AuthorizationDecreaseRequested` event we need the `decreasingAt` property, that is only passed from app's contract event, and is not present in `TokenStaking` contract event. We don't need the exact value, but it is needed to check if deauthorization request is active (see https://github.com/threshold-network/token-dashboard/blob/5875b44992cbe0c24439d227c6ca89e0568b11ca/src/store/staking-applications/slice.ts#L208-L219).

Do you know if it's somehow possible to get that infrormation from `TokenStaking` contract? 

For now @kpyszkowski has only changed the `AuthorizationIncrease` event as it was stated in the issue.

**Comment by cygnusv** [comment]
> Do you know if it's somehow possible to get that infrormation from `TokenStaking` contract?

I don't think so (although please @vzotova correct me if I'm wrong), since that information seems specific to the sortition pool, not the staking contract. I see 2 alternatives:
* It seems that, roughly, `decreasingAt = operatorIsUnknown? block.timestamp : MAX_UINT64`, so if you can get a way to query if the operator was known or not from the sortition pool or tBTC app contract, then it's possible to try to replicate it off-chain (with the caveat that the original event is giving you that info for a particular block number)
* If I'm not wrong, in the same TX that fires the staking contract events, there will be the app events, so you can use both to complement each others' information. Using the app events is OK, as long as it's clear that it's a tbtc app-specific need; in particular, TACo won't have the same app events as tBTC since there's no common interface for the events you mention.
